app/src/main/java/com/codesoom/assignment/application/MemberService.java

```diff
@@ -0,0 +1,40 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.MemberNotFoundException;
+import com.codesoom.assignment.domain.Member;
+import com.codesoom.assignment.domain.MemberRepository;
+import com.codesoom.assignment.dto.MemberData;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+    private final MemberRepository memberRepository;
+
+    public Member create(MemberData memberData) {
+        Member createdMember = Member.builder()
+                .name(memberData.getName())
+                .phone(memberData.getPhone())
+                .build();
+        memberRepository.save(createdMember);
+        return createdMember;
+    }
```

Comment:
```suggestion
    public Member create(MemberData memberData) {
        Member member = Member.builder()
                .name(memberData.getName())
                .phone(memberData.getPhone())
                .build();
        return memberRepository.save(member);
    }
```

JPA로 생성된 멤버를 반환하는게 좋을 것 같습니다. 이 멤버에는 식별자가 포함되어 있으니까요

Reply:
레포지터리에서 반환된 값을 이용하여 반환할 수 있도록 수정하겠습니다!